utils/dataloaders.py

```diff
@@ -65,29 +73,40 @@ def get_fashion_mnist_dataloaders(batch_size=128, shuffle=True,
     return train_loader, test_loader
 
 
-def get_dsprites_dataloader(batch_size=128, shuffle=True,
-                            path_to_data=os.path.join(DIR, '../data/datadsprites/dsprites_data.npz')):
+def get_dsprites_dataloader(batch_size=128,path_to_data=os.path.join(DIR, '../data/dsprites-dataset'), shuffle=True):
     """DSprites dataloader."""
-    dsprites_data = DSpritesDataset(path_to_data,
+    print('in loop  ')
+    root = os.path.join(os.path.dirname(DIR), 'data', 'dsprites-dataset')
+    if os.path.isdir(root)==False:
```

Comment:
don't test values of booleans. This should be `not os.path.isdir(root)`. please change it everywhere you do it